move-vm-backend-common/tests/assets/move-projects/signer_scripts/sources/Generics.move

```diff
@@ -0,0 +1,48 @@
+script {
+    use std::signer;
+
+    // Should fail!
```

Comment:
Just a random thought? Should we add in the descriptions the reason, why it should fail? :thinking: For better understanding? Or is that too much? What do you think?

Reply:
Maybe it is better to write the rules in one place - and then, if somebody wonders why this specific test should fail - they can refer to our rule set.

Maybe in the function description? Wdyt?

Reply:
Done